core/src/main/java/com/avast/bytes/AbstractBytesComparator.java

```diff
@@ -0,0 +1,35 @@
+package com.avast.bytes;
+
+
+import java.util.Comparator;
+
+public abstract class AbstractBytesComparator
+        implements Comparator<Bytes> {
+
+    private final boolean shorterFirst;
+
+    protected AbstractBytesComparator() {
+        this(true);
+    }
+
+    /**
+     * @param shorterFirst If true, shorter Bytes with same prefix will be take as less
```

Comment:
How about calling this `shorterIsLess`? By the way, how can you change this value? It's in protected constructor and the subclasses do not allow you to change it. Is that ok?